types/bull-arena/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
```

Comment:
What is the reason for having a `package.json` file?

Reply:
This file is required for bee-queue dependency. Otherwise, npm test was throwing this error.
`Cannot find module 'bee-queue' or its corresponding type declarations.`

Reply:
`bee-queue` is a public npm module. I don't think you need this file but I don't know. I could be wrong.

Reply:
It is required for passing tests. `bee-queue` is a dependency of `bull-arena`, so all users will have bee-queue installed.